Extensions/CMake/SlicerFunctionGenerateExtensionDescription.cmake

```diff
@@ -143,7 +148,8 @@ function(slicer_generate_extension_description_test)
     EXTENSION_DESCRIPTION "The SlicerToKiwiExporter module provides Slicer user with any easy way to export models into a KiwiViewer scene file.
 This is a line of text.<br>And another one."
     EXTENSION_CATEGORY "Exporter"
-    EXTENSION_CONTRIBUTORS "Jean-Christophe Fillion-Robin (Kitware), Pat Marion (Kitware), Steve Pieper (Isomics), Atsushi Yamada (Shiga University of Medical Science)"
+    EXTENSION_CONTRIBUTORS "Jean-Christophe Fillion-Robin (Kitware), Pat Marion (Kitware), \
```

Comment:
It is OK to enable this syntax, but I find it a bit complicated. Could standard CMake-style lists syntax be used here?

```
  EXTENSION_CATEGORY "Exporter"
  EXTENSION_CONTRIBUTORS
    "Jean-Christophe Fillion-Robin (Kitware)"
    "Pat Marion (Kitware)"
    "Steve Pieper (Isomics)"
    "Atsushi Yamada (Shiga University of Medical Science)"
  EXTENSION_HOMEPAGE "https://www.slicer.org/w/index.php/Documentation/Nightly/Extensions/SlicerToKiwiExporter"
```

Reply:
Ditto. That would indeed be sensible, an issue was created to capture this.